README.md

```diff
@@ -1,3 +1,5 @@
 # apollo-fetcher
 
-This is the beginning of apollo-fetcher. The new network interface layer for Apollo Client.
+apollo-fetcher is a standard interface for fetching GraphQL results, designed to support the network interface of Apollo Client and provide a simple standalone GraphQL client capable of extensions.
```

Comment:
I don't think we need to mention Apollo Client here at all. We can always add it later.